frontend/src/components/Answer/AnswerParser.tsx

```diff
@@ -38,6 +38,16 @@ export function removeCitations(text: string): string {
     return text.replace(RX_CITATION_BLOCK, "");
 }
 
+/**
+ * Remove the Citations block and its links.
+ */
+export function removeCitationsBlock(input: string): string {
```

Comment:
@ramirezmorac2 Does this change affect the outcome of the unit tests? Also, did you run the unit tests locally to verify? If so, wouldn’t it make sense to add this specific case to the test suite as well?